core/src/apps/monero/layout.py

```diff
@@ -143,6 +145,7 @@ async def _require_confirm_output(
     dst: MoneroTransactionDestinationEntry,
     network_type: MoneroNetworkType,
     payment_id: bytes | None,
+    chunkify: bool = False,
```

Comment:
remove default value

Reply:
Done in [7e03759](https://github.com/trezor/trezor-firmware/pull/3237/commits/7e03759ea7c3f0e2e56218ee1bd5cb18bef9c134)